pkg/jobstore/boltdb/store.go

```diff
@@ -1092,24 +1092,28 @@ func (b *BoltJobStore) updateExecution(tx *bolt.Tx, request jobstore.UpdateExecu
 		}
 	}
 
-	return b.appendExecutionHistory(tx, newExecution, existingExecution.ComputeState.StateType, request.Event)
+	return nil
+}
+
+// AddExecutionHistory appends a new history entry to the execution history
+func (b *BoltJobStore) AddExecutionHistory(ctx context.Context, jobID, executionID string, events ...models.Event) error {
+	return b.update(ctx, func(tx *bolt.Tx) (err error) {
```

Comment:
Is it possible to batch these operations?
https://pkg.go.dev/github.com/boltdb/bolt#DB.Batch